src/components/TaskDetails/TaskPriority.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Container, Dropdown } from 'react-bootstrap';
+
+export default class TaskPriority extends React.Component {
+  handleChangePriority = (priorityName) => {
+    const { changePriority } = this.props;
+    changePriority(false, false, false, priorityName);
+  };
+
+  render() {
+    const { priority } = this.props;
+    switch (priority) {
+      case 'High':
```

Comment:
i would suggest to add classes like `priority-${priority}` and describe different colors for different classes in css